### DISCUSSION TITLE
Environment Variables command line

### DISCUSSION THREAD

**Discussion by precator** [aquasecurity/trivy#957] [Q&A] [answered]
Can environment variables such as below be set on the command line when calling the Trivy executable?

export TRIVY_USERNAME={USERNAME}
export TRIVY_PASSWORD={PASSWORD}

like 
TRIVY_USERNAME={USERNAME} TRIVY_PASSWORD={PASSWORD} TRIVY_NON_SSL=true trivy image -f json -o`

**Comment by cww1769** [comment]
I what to know the answer too.

**Reply by knqyf263** [reply, marked as answer]
Here it is
https://aquasecurity.github.io/trivy/v0.41/docs/configuration/#environment-variables
